ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -109,6 +109,17 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         case custom
     }
 
+    @objc(MSFNavigationBarTitleStyle)
+    /// Describes the style in which the title is shown in a navigation bar.
+    public enum TitleStyle: Int {
+        /// Shows a center-aligned title and/or subtitle. Most closely aligned with UIKit's default. Not capable of showing an avatar. b
```

Comment:
nit: "Not capable of showing an avatar. b"

Reply:
The B stands for "bane of my existence, these typos are." But yeah, good catch.